project-base/scripts/install.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+numberRegex='^[0-9]+([.][0-9]+)?$'
+operatingSystem=''
+allowedValues=(1 2)
+declare -A operatingSystemsByIndex
+
+operatingSystemsByIndex[1]='Linux'
+operatingSystemsByIndex[2]='Mac'
+
+echo This is installation script that will install demo Shopsys Framework application on docker with all required containers and with demo database created.
+
+echo "Start with specifying your operating system: \
+    1) Linux
+    2) Mac
+    "
```

Comment:
Windows can be added because we develop on windows via WSL and docker-sync where we have files `docker-sync-win.yml` and `docker-compose-win.yml` so almost the same principle like for Mac